cluster-api/src/main/java/io/aiven/klaw/clusterapi/services/SchemaService.java

```diff
@@ -257,20 +248,77 @@ private String getSchemaCompatibility(
     }
   }
 
+  private Pair<String, String> getSubjectCompatibilityWithGlobalFallBack(
+      String environmentVal,
+      String topicName,
+      KafkaSupportedProtocol protocol,
+      String clusterIdentification) {
+    try {
+      if (environmentVal == null) {
+        return null;
+      }
+
+      String compatibility =
+          getSubjectSchemaCompatibility(environmentVal, topicName, protocol, clusterIdentification);
+
+      if (!compatibility.equals(SCHEMA_COMPATIBILITY_NOT_SET)) {
```

Comment:
possibility for npe

Reply:
Deleted Method as no longer required with change in logic.